src/displayapp/fonts/jetbrains_mono_bold_20.c

```diff
@@ -1,17 +1,17 @@
 /*******************************************************************************
  * Size: 20 px
  * Bpp: 1
- * Opts: 
+ * Opts:
```

Comment:
I had to regenerate this file because I added the "dice" symbol, and I avoided it when running clang-format, but I just realized that there's a git hook (hooks/pre-commit) that must have done this when I made my commit...  I'll fix this and look into filtering out certain files from that git hook

Reply:
I resolved this in the latest changes.